gitless/cli/gl_branch.py

```diff
@@ -149,8 +149,16 @@ def _do_create(create_b, dp, repo):
           continue
         remote_str = ' in remote repository {0}'.format(maybe_remote)
     try:
-      r.create_branch(b_name, target)
+      new_branch = r.create_branch(b_name, target)
       pprint.ok('Created new branch {0}{1}'.format(b_name, remote_str))
+      try:
+          remote, branch = dp.split('/', 1) if '/' in dp else ('' , '')
```

Comment:
You don't need to do the split yourself. Take a look at the code of `get_branch`. It will check if the branch is a remote branch automatically:

https://github.com/sdg-mit/gitless/blob/master/gitless/cli/helpers.py#L23
https://github.com/sdg-mit/gitless/blob/master/gitless/cli/helpers.py#L30